### DISCUSSION TITLE
Side Panel Ingress

### DISCUSSION THREAD

**Discussion by LordNex** [hassio-addons/app-bookstack#186] [Q&A] [answered]
Would it be possible to add a side panel ingress so that Home Assistant users can view documents without having to leave Home Assistant?

**Comment by frenck** [comment, marked as answer]
That is not possible with this add-on, as the application wrapped in this add-on (Bookstack) won't allow for it (technical limitation upstream).

**Reply by LordNex** [reply]
Ahh I figured it was something like that. I really need to sit down with someone who knows python well and have them explain it too me. I've been in IT for over 20 years and know my stuff. But when I start getting into code I get lost. There are just some fundamental things I think I'm just not understanding or hasn't been explained right too me. 

But I can build a SAN array clustered in a VMWare VSphere network and provision everything through Microsoft System Center for you to run it on! LOL we all have our talents.
